docs/en/ingest-management/tab-widgets/upgrade-fleet.asciidoc

```diff
@@ -0,0 +1,81 @@
+// tag::ess[]
+
+{ecloud} runs a hosted version of {fleet-server}.
+
+To confirm that {fleet-server} is available in your deployment:
+
+. Log in to {kib} and go to *Management > {fleet}*, and click the *Agents* tab.
+. The following message is displayed. Please note that your {agent}s have now
+been unenrolled and have stopped sending data. 
++
+Select *Do not show this message again*, and then click *Close and get started*.
++
+[role="screenshot"]
+image::images/fleet-server-prompt.png[Add {fleet-server} prompt]
++
+If your previous deployment did not include an APM node, you are prompted to enable *APM & {fleet}*,
+required for using {fleet-server}.
++
+Click *Edit deployment* and add an *APM & Fleet* node to your deployment.
++
+[role="screenshot"]
+image::images/apm-fleet-prompt.png[Add APM & Fleet node prompt]
++
+. To confirm that {fleet-server} is available in your deployment, click the *Agents* tab.
+. Under *Policies*, select the *Elastic Cloud agent policy* to confirm that {fleet-server}
+is listed. This policy is managed by {ecloud} and can not be modified.
++
+You now need to install and enroll your agents.
+
+// end::ess[]
+
+// tag::self-managed[]
+
+To upgrade to {fleet-server}:
+
+//TODO: Mention API for adding the token.
+
+. Log in to {kib} and go to *Management > {fleet}*.
+. Click the *Agents* tab.
+. The following message is displayed. Please note that your {agent}s have now
+been unenrolled and have stopped sending data.
++
+Select *Do not show this message again*, and then click *Close and get started*.
++
+[role="screenshot"]
+image::images/fleet-server-prompt-managed.png[Add {fleet-server} prompt]
+
+. Click *{fleet} settings*, and in the *{fleet-server} hosts* field, enter the
+URLs {agent}s will use to connect to {fleet-server}. For example,
+`https://192.0.2.1:8220`, where `192.0.2.1` is the host IP where you will
+install {fleet-server}.
+
+. In the *{es} hosts* field, specify the {es} URLs where {agent}s will send data.
+For example, `https://192.0.2.0:9200`.
+
+. Save and apply the settings.
+
+. Click the *Agents* tab and follow the in-product instructions to add a
+{fleet} server:
++
+[role="screenshot"]
+image::images/add-fleet-server.png[In-product instructions for adding a {fleet-server}]
++
+The `install` command installs the {agent} as a managed service and enrolls it
+in a {fleet-server} policy:
++
+[source,yaml]
+----
+sudo ./elastic-agent install -f --fleet-server-es=http://localhost:9200 \
+--fleet-server-service-token=AAEAAWVsYXN0aWMvZmxlZXQtc2VydmVyL3Rva2VuLTE2MTkxMzg3MzIzMTg6dzEta0JDTmZUZGlDTjlwRmNVTjNVQQ
+----
++
```

Comment:
```suggestion
Note, for .deb or .rpm artifact usage, substitute the 'install' command for 'enroll'.  
This is because .deb and .rpm usage inherently include installing the {agent} files to disk and so the 'install' command is incompatible with them.
```